BackEnd/src/routes/api/task.js

```diff
@@ -49,19 +50,32 @@ createRoutes([
                     { key: 'taskId',        type: 'number' },
                     { key: 'title',         type: 'string', optional: true },
                     { key: 'description',   type: 'string', optional: true },
-                    { key: 'course',        type: 'string', optional: true }, // Note: password can't consist of only numbers using this method
                     { key: 'deadline',      type: 'string', optional: true },
-                    { key: 'graded',        type: 'boolean',optional: true },
                 ]);
+
+                // TODO: check deadline is date, if exists
+
+                const dbArgs = {
+                    Title: args.title,
+                    Description: args.description,
+                    Date: args.deadline,
+                }
+                // get only the keys which have a value assigned
+                const validKeys = Object.keys(dbArgs).filter(key => 
```

Comment:
Könntest du hier nicht auch einfach einmal drüber loopen und abfragen, ob das value undefined ist und wenn nicht das value in options übernehmen? Ich weiß ehrlich gesagt nicht ob das dann sauberer wäre, aber ist ne Idee die mir gekommen ist :)

Reply:
Mach ich :+1: